infra/integration-image/git-clone.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+sed -i 's|remotes/origin/*|remotes/origin/*\n\tfetch = +refs/pull/*/head:refs/remotes/origin/pr/|g' .git/config
+git fetch origin && git checkout pr/$DRONE_PULL_REQUEST
```

Comment:
If I'm not wrong, Drone currently checks out merge commit of a PR. 
`git fetch --tags origin +refs/pull/$DRONE_PULL_REQUEST/merge`
This changes it to check out the PR branch alone. Is this intentional change?

Reply:
yes, the merge url it checks out is unsupported and is what causes us to see the build mismatch errors